inc/options/general.php

```diff
@@ -481,6 +481,17 @@ class="w3tc-ignore-change" type="text"
 				</th>
 			</tr>
 			<?php
+			if ( version_compare( get_bloginfo( 'version' ), '5.5', '<' ) ) {
```

Comment:
So we're adding a new setting for users to disable jquery migrate on the front end? I wasn't really aware what was in this ticket.

Reply:
Yes, see the link in this PR.  I removed the version check.  https://make.wordpress.org/core/2020/06/29/updating-jquery-version-shipped-with-wordpress/